### PR TITLE
ign -> gz Macro Migration : gz-physics

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

File: include/ignition/physics/OperateOnSpecifiedData.hh [hunk 1]
```diff
@@ -17,3 +17,8 @@
 
 #include <gz/physics/OperateOnSpecifiedData.hh>
 #include <ignition/physics/config.hh>
+
+#define IGN_PHYSICS_OPERATEONSPECIFIEDDATA_TEMPLATES \
+  GZ_PHYSICS_OPERATEONSPECIFIEDDATA_TEMPLATES
```
Comment (chapulina) [inline include/ignition/physics/OperateOnSpecifiedData.hh:22]: These are defined in the `detail` directory, can we skip the tick-tock?

File: Migration.md [hunk 2]
```diff
@@ -14,6 +14,24 @@ release will remove the deprecated code.
...
+   1. `IGN_PHYSICS_MAKE_ALL_TYPE_COMBOS`
+   1. `IGN_PHYSICS_ADD_PLUGIN`
+   1. `IGN_PLUGIN_REGISTER_MORE_TRANS_UNITS` -> `GZ_**PHYSICS**_REGISTER_MORE_TRANS_UNITS`
+   1. `IGN_PHYSICS_REQUEST_FEATURES_MACRO`
+   1. `IGN_PHYSICS_CREATE_SELECTOR`
+   1. `IGNITION_PHYSICS_CONST_GET_ERROR` (hard-tocked, inside detail header)
+   1. `IGNITION_UNITTEST_EXPECTDATA_ACCESS` (hard-tocked, inside test and detail headers)
+   1. `IGNITION_PHYSICS_DEFINE_COORDINATE_SPACE` (hard-tocked, inside detail header)
+   1. `IGN_PHYSICS_MOCK_MACRO_HELPER`
+   1. `IGN_PHYSICS_MOCK_MACRO`
```
Comment (chapulina) [inline Migration.md:32]: These `MOCK` macros aren't being tick-tocked, they're not exposed to users, so no need to add to the migration guide

Comment (methylDragon) [reply Migration.md]: https://github.com/gazebosim/gz-physics/pull/353/commits/c18e0f04d406ad066d91dbca97f2f4af63e16def

File: include/ignition/physics/OperateOnSpecifiedData.hh (hunk 1 above)
Comment (methylDragon) [reply include/ignition/physics/OperateOnSpecifiedData.hh]: https://github.com/gazebosim/gz-physics/pull/353/commits/c18e0f04d406ad066d91dbca97f2f4af63e16def :boom:

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**